src/inspirehep-holdingpen-js/templates/inspirehep-holdingpen-js/results/results.html

```diff
@@ -108,19 +106,23 @@
 
             <div
               ng-if="record._source._workflow.data_type === 'hep' && !record._source.wo.user_action && record._source._workflow.status === 'HALTED' && record._source._extra_data._action === 'hep_approval' && !hasConflicts(record)">
-                <button class="btn btn-success"
+                <span ng-if="record._source.metadata.inspire_categories.length">
```

Comment:
it's better to add a ``div`` instead

Reply:
Yeah, but if  I do that, the reject button will not be displayed in the same line as the core and accept ones

Reply:
```display: 'inline'```